### ISSUE TITLE
OnValueChanging is triggered a second time after OnValueChanged on custom question type

### ISSUE THREAD

**Issue by PhilippeKayser** [surveyjs/survey-library#4289]
### Are you requesting a feature, reporting a bug or asking a question?

Bug

### What is the current behavior?

OnValueChanging is triggered an additional time after OnValueChanged on custom question types

- onValueChanging is triggered

|  (index) |   Values  |
|:--------:|:---------:|
| oldValue | undefined |
| value    | 'abc      |

- onValueChanged is triggered

- onValueChanging is triggered again

|  (index) |   Values  |
|:--------:|:---------:|
| oldValue | 'abc' |
| value    | 'abc      |

### What is the expected behavior?

OnValueChanging is only triggered once before OnValueChanged

### How would you reproduce the current behavior (if this is a bug)?

Please refer to the [this repository](https://github.com/PhilippeKayser/surveyjs-custom-questiontype-bug) with a simple reproduction of the bug. I added one use case of known type *text* and one for a custom type with it's questionJSON being of type *text* for comparison.

I am currently overriding this behaviour with the following piece of code:

```    javascript
valueChangingHandler = (sender, options) => {
    if(options.oldValue === options.value) {
        console.warn('onValueChanging execution with equal values', options.question, options.oldValue, options.value)
        return;
    }
    ...
} 
```

### Specify your

I am using survey-core as a backbone for a custom survey system.

* browser: /
* browser version: /
* surveyjs platform (angular or react or jquery or knockout or vue): /
* surveyjs version: **1.9.27** (most recent)

**Comment by andrewtelnov** [comment]
@PhilippeKayser Unfortunately, I could not reproduce this behaviour. Here is [the plunker](https://plnkr.co/edit/Ao1v95HG5ocT3mbN) that works correctly to me.
Probably there is an issue in a particular custom widget that works incorrectly.
Anyway, feel free to fork the plunker, modify it, reproduce the issue and post a new link here. We will take a look.

Thank you,
Andrew

**Comment by PhilippeKayser** [comment]
@andrewtelnov Hi Andrew, thank you very much for your reply. Indeed, your plunker works and, weirdly enough, if I change it so it reflects our problem, it works as well. But using it with one of the surveyjs frontends is not our use-case.

As mentioned, I use a completely custom interface, so I only use the core dependency, nothing else. Here, the bug occurs when I assign a value to a question of a custom question type in code, added via Survey.ComponentCollection.Instance.add(...). You can see in [this plunker](https://plnkr.co/edit/Qfxp1QdD1U7z2DDZ), which really is the best simplification of this behaviour I could find, that my custom type is just a wrapper around a question of type text. When assigning the question with a value in code, it triggers onValueChanging a second time after onValueChanged. Can you please check it out and confirm that a second valueChanging event is fired? Thank you!

**Comment by andrewtelnov** [comment]
@PhilippeKayser I have fixed by the commit above. My PR should be approved shortly and the fix will be available in [v1.9.28](https://surveyjs.io/WhatsNew#v1.9.28).

Thank you,
Andrew
